src/vl-map-layer-style.js

```diff
@@ -82,17 +139,51 @@ export class VlMapLayerStyle extends vlElement(HTMLElement) {
    * @Return {string}
    */
   get style() {
-    return new OlStyle({
+    return (feature) => {
+      const styleConfig = {
+        fill: new OlStyleFill({
+          color: this.color,
+        }),
+        stroke: new OlStyleStroke({
+          color: this.borderColor,
+          width: this.borderSize,
+        }),
+      };
+      if (this.labelFunction != null) {
+        styleConfig['text'] = this._getTextStyle(feature);
+      }
```

Comment:
Is het nodig om te controleren op `labelFunction`? Wat gebeurt er als je de OlStyleText toevoegt en de labelFunctie null geeft? Als dit geen effect heeft, zou ik die stijl default toevoegen.

Reply:
Nee hoeft op zich niet, maar was dan minder achterliggende code, maar niet persé nodig, heb het weggehaald.